nexus/src/external_api/tag-config.json

```diff
@@ -103,12 +103,6 @@
       "external_docs": {
         "url": "http://docs.oxide.computer/api/system-silos"
       }
-    },
-    "system/update": {
-      "description": "This includes operations related to system updates.",
-      "external_docs": {
-        "url": "http://docs.oxide.computer/api/system-update"
-      }
```

Comment:
Thought about leaving this in, but if not I would have to update the docs site to handle empty tags. The tags test will remind us to put it back in when we re-publish these endpoints.